src/svg/SvgImport.js

```diff
@@ -437,7 +437,9 @@ new function() {
     }, {}), {
         id: function(item, value) {
             definitions[value] = item;
-            if (item.setName)
+            // Same check used in setName to test if IDs are numeric. Numeric IDs make setName throw an exception.
+            // Better to duplicate the check than to wrap this in a try block & swallow all other potential exceptions.
+            if (item.setName && ((+value) + '' !== value))
                 item.setName(value);
```

Comment:
Maybe we should add a letter to it and set name instead?

Max brings up that this could cause name collisions

Reply:
https://github.com/paperjs/paper.js/blob/b081fd72c72cd61331313c3961edb48f3dfaffbd/src/item/Item.js#L325-L329 for reference

Reply:
We can also use random ID.